src/ModalManager.js

```diff
@@ -75,6 +79,11 @@ class ModalManager extends React.Component {
     return this.props.onDone();
   }
 
+  shouldWithdrawnModalBeShown = () => {
+    const { checkedinItem } = this.state;
+    return get(checkedinItem, 'status.name') === statuses.WITHDRAWN;
```

Comment:
Do you want to use `checkedinItem?.status?.name` for this?

Reply:
@Baroquem thank you for catching this!